### DISCUSSION TITLE
option to specify the run template directory

### DISCUSSION THREAD

**Discussion by ltfafei** [projectdiscovery/nuclei#3263] [Q&A] [answered]
It is recommended that you add the option to specify the run template directory，Discover The -t option can specify only one or more template files, but cannot specify the template directory。

**Comment by ehsandeep** [comment, marked as answer]
> Discover The -t option can specify only one or more template files, but cannot specify the template directory。

@ltfafei it's already supported, `-t cves` runs all the templates under cves directory.

**Comment by ltfafei** [comment]
Allright ~
